### ISSUE TITLE
Models arn't combined into 1 object but separate mesh files in UE4 

### ISSUE THREAD

**Issue by Heagan** [code4game/glTFForUE4#51]
I'm trying this with a GLTF File with 3800 pieces generated automatically from another program

Its a map model, so the tree's and ground are being treated as meshes 

I was hoping to have them combined into 1 mesh object like when imported in Blender or Godot

**Comment by jixingcn** [comment]
Yes, I know that. Will add this as a feature in the next version.

**Comment by kosenhitatchi** [comment]
Hi folks.
I hope you're all doing great.
First, thanks @jixingcn and @alexchicn for your amasing work on this plugin.

In my current use-case I'm trying to get a .glb from [Ready Player Me](https://readyplayer.me), and your plug-in is actually doing a pretty nice job (much better than unreal build-in importer). Well as expected it do not output a file with all the mesh combined.
Thus, as the asset from there are animated and composed of several mesh (VR faces asset) I think it could be a nice addition to your test.

All the best

**Comment by jixingcn** [comment]
Glad to hear your said. Thanks. @kosenhitatchi

All meshes were combined in one static mesh with sections if you used the first version of the plugin. Now, I think it can be an option that allows users to decide to combine or not by themself.
